src/porepy/numerics/linalg/linsolve.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Various functionality for linear solvers.
+
+Created on Thu Nov  2 19:08:28 2017
+
+@author: Eirik Keilegavlen
+"""
+import numpy as np
+import scipy.sparse.linalg as spl
+
+try:
+    import pyamg
+except ImportError:
+    " Could not import the pyamg package. pyamg sover will not be available."
```

Comment:
Typo, second sentence

Reply:
Yes, they are part of the search for a decent solver. To be included, I hope, if not then purged later.